Source/JavaScriptCore/dfg/DFGGraph.h

```diff
@@ -1176,6 +1177,14 @@ class Graph final : public virtual Scannable {
     Vector<BasicBlock*, 1> m_roots;
     Vector<Edge, 16> m_varArgChildren;
 
+    struct TupleData {
+        uint16_t refCount { 0 };
+        uint16_t resultFlags { 0 };
+        VirtualRegister virtualRegister;
+    };
```

Comment:
You don't really need the prediction here because consumers of the node will see the prediction via the ExtractFromTuple. I could see adding OpInfos here for extra metadata as needed but it seems like we should do that when we need it.